# PPIN-Nevi-vs-Melanoma
[![DOI](https://zenodo.org/badge/849765878.svg)](https://doi.org/10.5281/zenodo.13903581)


![TOC_Final](https://github.com/user-attachments/assets/df8b58c6-b8ad-4c06-abe6-366aacf527d8)


Analysis of protein-protein interactions (PPIs) may provide deeper insight into protein functions and reveal how components of cellular pathways interact with each other. There are multiple software tools that infer condition-specific PPI networks based on transcriptomic data. However, it is well known that the transcriptome does not always reflect the proteome. To address this, we present here a new computational approach that is able to characterize rewiring events in PPI networks based on proteomic data exclusively. We applied this approach by comparing a cohort of 14 nevi samples to their corresponding nevus-associated melanomas. Thereby, we were able to identify differentially expressed proteins, deregulated protein interactions, protein clusters and hub proteins that were consistent with previously published evidence on the progression of nevi to melanoma. Additionally, we compared the proteomics data against matched spatial transcriptomics data (GeoMX data for 8 out of 14 samples). This comparison highlighted the different outcomes of each approach and further emphasize the importance of considering multiomics analysis.
